viral/learning_stages.py

```
@@ -127,6 +127,7 @@ def main() -> None:
             mouse_name, SESSIONS_KEEP[mouse_name]["learned"], metadata, stage="learned"
         ),
     )
+    1 / 0
```

Ruff (0.11.9)

Found useless expression. Either assign it to a variable or remove it.

(B018)